### PR TITLE
Apollo Server quickstart dashboard

### PR DESCRIPTION
**Pull request by dmiyamasu**
# Summary

As part of the great dashboard bash, I have created a dashboard for the Apollo-server quick start

## Pre merge checklist

- [ ] Did you check you NRQL syntax? - Does it work?
- [ ] Did you include a Data source and Documentation reference?
- [ ] Are all documentation links publicly accessible?
- [ ] Did you check your descriptive content for [voice and tone](https://docs.newrelic.com/docs/style-guide/writing-strategies/voice-strategies-docs-sound-new-relic/)?
- [ ] Did you check your descriptive content for spelling and grammar errors?
- [ ] Did you review your content with a subject matter expert? (e.g. a Browser agent quickstart is reviewed with a member of the Browser Agent team)

### Dashboards

- [X] Does the PR contain a screenshot for each of your dashboards?
- [x] Do your screenshots show data?
- [ ] Has the [sanitization script](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#dashboards) been run on each dashboard?

### Alerts

- [ ] Did you check that your alerts actually work?

### CODE REVIEW

File: dashboards/apollo-server/apollo-server.json [hunk 1]
```diff
@@ -0,0 +1,191 @@
+{
+    "name": "Apollo-Server Dashboard",
+    "description": null,
+    "pages": [
+      {
+        "name": "Apollo-Server Dashboard",
+        "description": null,
+        "widgets": [
+          {
+            "title": "Top 10 Slowest Operations",
```
Comment (seemantk) [inline dashboards/apollo-server/apollo-server.json:10]: @dmiyamasu would it be useful to also show the top 10 fastest operations?

File: dashboards/apollo-server/apollo-server.json [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+              "units": {
+                "unit": "SECONDS"
+              },
+              "yAxisLeft": {
+                "zero": true
+              }
+            }
+          },
+          {
+            "title": "Top 10 Slowest Resolvers",
```
Comment (seemantk) [inline dashboards/apollo-server/apollo-server.json:100]: @dmiyamasu ditto here, would it be useful to have top 10 resolvers?

**Review by caylahamann — COMMENTED** [review]
This looks good to me from a technical standpoint - I'll defer to @seemantk for reviewing content

File: dashboards/apollo-server/apollo-server.json (hunk 1 above)
Comment (seemantk) [reply dashboards/apollo-server/apollo-server.json:10]: Thank you for clarifying @dmiyamasu.

**Comment by seemantk** [comment]
LGTM, thank you @caylahamann please feel free to proceed with merging this

**Review by aswanson-nr — APPROVED** [review]

**Pull request merged**